<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cropyield</class>
 <widget class="impgenmaint" name="cropyield">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="header1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Household information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="header2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Household information</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Label33">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Yield (kg/plot):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IMPEdit" name="txtyield">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>yield</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="IMPEdit" name="txtfs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>porfs</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="Label28">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="IMPCheckBox" name="chkhc">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Human consumption</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>hc</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="IMPEdit" name="txthc">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>porhc</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="Label29">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="IMPCheckBox" name="chklf">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Livestock feeding</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>lf</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="IMPEdit" name="txtlf">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>porlf</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="Label30">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="IMPCheckBox" name="chklv">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Leave on the field</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>le</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="IMPCheckBox" name="chkfs">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>For sale</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>fs</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="IMPEdit" name="txtlv">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>porle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="Label31">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="IMPCheckBox" name="chkmm">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Manure management</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>mm</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="IMPEdit" name="txtmm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>pormm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="Label32">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="IMPCheckBox" name="chkst">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Store</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>st</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="IMPEdit" name="txtst">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>porst</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="Label37">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="IMPCheckBox" name="chkot">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Other</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="FieldName">
        <string>ot</string>
       </property>
       <property name="CheckedValue" stdset="0">
        <string>Y</string>
       </property>
       <property name="UnCheckedValue">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="IMPEdit" name="txtot">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="FieldType">
        <enum>IMPEdit::Numeric</enum>
       </property>
       <property name="FieldName">
        <string>porot</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="Label54">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>kgs</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BitBtn19">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Apply this information to all the selected months</string>
       </property>
       <property name="text">
        <string>Apply to all periods</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/apply.png</normaloff>:/images/apply.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BitBtn16">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/apply.png</normaloff>:/images/apply.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BitBtn17">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BitBtn18">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/closeform.png</normaloff>:/images/closeform.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IMPEdit</class>
   <extends>QLineEdit</extends>
   <header>impedit.h</header>
  </customwidget>
  <customwidget>
   <class>IMPCheckBox</class>
   <extends>QCheckBox</extends>
   <header>impcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>impgenmaint</class>
   <extends>QWidget</extends>
   <header>impgenmaint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
